helm/panda/values.yaml

```diff
@@ -30,9 +30,22 @@ server:
 
   # use Route class for ingress
   route:
-    enabled: true
+    enabled: false
     # host name of the PanDA server
     hostName: "FIXME"
+  ingress:
+    enabled: true
+    hosts:
+      - domain: cern.ch
```

Comment:
The top values.yaml is the file that the user mainly customizes. Can you put "FIXME" instead of a cluster-specific string like "cern.ch" so that the user can clearly see which params needs to be set?

Also I see top values.yaml of idds and  activemp are quite messy. Do you really need to expose all parameters there?  It wold be enough to put only a minimum set of params that have to be always set for each deployment since default values in each chars/*/values.yaml work out of the box in many cases.